tools/integration_tests/local_file/helpers/file_operations_helper.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2023 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package helpers
+
+import (
+	"os"
+	"strings"
+	"testing"
+)
+
+func ValidateNoFileOrDirError(path string, t *testing.T) {
+	_, err := os.Stat(path)
+	if err == nil || !strings.Contains(err.Error(), "no such file or directory") {
+		t.Fatalf("os.Stat(%s). Expected: %s, Got: %v", path,
+			"no such file or directory", err)
+	}
+}
+
+func VerifyRenameOperationNotSupported(err error, t *testing.T) {
```

Comment:
this is specific to rename. please move this to rename_test.go

Reply:
done